debian/control

```diff
@@ -0,0 +1,46 @@
+Source: seafile-gui
+Section: net
+Priority: extra
+Maintainer: m.eik michalke <meik.michalke@hhu.de>
+Build-Depends:
+    debhelper (>= 7),
+    dh-python,
+    python,
```

Comment:
aren't the two python scripts in the scripts directory (build_helper.py and build.py) used for the build process?